src/D3MHub.sol

```diff
@@ -331,6 +329,17 @@ contract D3MHub {
         emit Unwind(ilk, amount, fees);
     }
 
+    function _fix(bytes32 ilk, address _pool) internal returns (uint256 diff) {
```

Comment:
Should we explore the idea to have multiple ways to enter this function to avoid loading the vat.urns multiple times? I don't think it is worth it due gas is not sensitive here but just want to make sure we agree on this.

Reply:
I noticed this gas optimization, but didn't comment because:

1. These functions are not gas sensitive. Infrequently called for large sums of money. The gas is trivial.
2. The SLOAD is already hot, so it's not that bad.